docs/docs/overview/quickstart/get-started-with-dotnet.md

```diff
@@ -0,0 +1,359 @@
+---
+sidebar_position: 11
+sidebar_label: Get started with .NET
+---
+
+# .NET Quickstart
+
+Learn how to use Novu to quickly send multi-channel (SMS, Email, Chat, Push) notifications from a .NET app.
+
+In this Quickstart, you’ll learn how to:
+
+- Install and use the Novu .NET SDK via the dotnet CLI.
+- Configure a Notification Channel Provider.
+- Set up a notification template.
+- Create subscribers and update subscriber information.
+- Send your first notification.
+- Send notifications via topics.
```

Comment:
Please link these points to their respective parts on the page. For example

[Install and use the Novu .NET SDK via the dotnet CLI](#install-novu-net-sdk)